package.json

```diff
@@ -2,86 +2,86 @@
   "name": "pentf",
   "version": "1.0.95",
   "description": "parallel end-to-end test framework",
-  "main": "index.js",
-  "types": "index.d.ts",
+  "main": "src/index.js",
+  "types": "types/index.d.ts",
```

Comment:
Is this correct? The clean task doesn't touch this, only `docs` and `dist`, and the `tsconfig.json` change references `"declarationDir": "dist/types/",`

Reply:
Good catch! You're right, it should be `dist/types/index.d.ts` 👍